src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1025,6 +1026,12 @@ struct SmartAction
             uint32 wp6;
         } closestWaypointFromList;
 
+        struct
+        {
+            uint32 sound[5];
```

Comment:
@Richet2
uint32 sound[5] - sets the array to have 5 elements
for (uint8 i = 0; i < SMART_ACTION_PARAM_COUNT - 1; i++) - is checking the array with indexes 0, 1, 2, 3, 4 (all together 5 elements)

isn't it correct?